client/src/components/Members.js

```diff
@@ -49,17 +63,50 @@ class Members extends Component {
     });
   }
 
+  //react-select documentation https://react-select.com/props
   render() {
     return (
-      <form onSubmit={this.handleSubmit}>
-        <h1>Members Page</h1>
-        <p>Gender ID: <input type='text' name='genderID' value={this.state.genderID} onChange={this.handleChange} /></p>
-        <p>Min Age: <input type='text' name='minAge' value={this.state.minAge} onChange={this.handleChange} /></p>
-        <p>Max Age: <input type='text' name='maxAge' value={this.state.maxAge} onChange={this.handleChange} /></p>
-        <p>Family Status ID: <input type='text' name='familyStatusID' value={this.state.familyStatusID} onChange={this.handleChange} /></p>
-        <p>Max Monthly Budget: <input type='text' name='maxMonthlyBudget' value={this.state.maxMonthlyBudget} onChange={this.handleChange} /></p>
-        <input type='submit'></input>
-      </form>
+      <div>
+        <form onSubmit={this.handleSubmit}>
+          <h1>Members Page</h1>
+          <p>Gender ID: <Select
+            isMulti
+            name="genderID"
+            options={[
+              { value: 'Male', label: 'Male' },
+              { value: 'Female', label: 'Female' },
+              { value: 'Other', label: 'Other' }
+            ]}
+            value={this.state.genderID}
+            onChange={this.handleChange}
+            isClearable true
+            className="basic-multi-select"
+            classNamePrefix="select"
+          /></p>
```

Comment:
I don't think all these menu options should be nested in `<p>` tags - in fact, I seem to get errors in the console when I run this in the browser. If you didn't know, when testing your code you can press F12 in your browser to open up the debug window and check for errors in the console tab. It's definitely saved my life a few times!